### PR TITLE
[DOCS] Release notes for 7.7.1 (bugfix release)

### PR DESCRIPTION
**Pull request by williamrandolph**
These are the release notes for the 7.7.1 bugfix release. I have been having some trouble with building locally so I may be doing some troubleshooting on the PR.

### CODE REVIEW

File: docs/reference/release-notes/7.7.asciidoc [hunk 1]
```diff
@@ -1,21 +1,141 @@
...
-[discrete]
+[[enhancement-7.7.1]]
+[float]
+=== Enhancements
+
+Authorization::
+* Remove synthetic role names of API keys as they confuse users {pull}56005[#56005]
+
+Features/ILM+SLM::
+* ILM add cluster update timeout on step retry {pull}54878[#54878]
```
Comment (jrodewig) [inline docs/reference/release-notes/7.7.asciidoc:14]:
```suggestion
* ILM: Add cluster update timeout on step retry {pull}54878[#54878]
```

File: docs/reference/release-notes/7.7.asciidoc [hunk 2]
```diff
@@ -1,21 +1,141 @@
...
+
+SQL::
+* SQL: change error message for comparison against fields in filtering {pull}57126[#57126] (issue: {issue}57005[#57005])
```
Comment (jrodewig) [inline docs/reference/release-notes/7.7.asciidoc:17]: This prefix is redundant as the heading is identical.
```suggestion
* Change error message for comparison against fields in filtering {pull}57126[#57126] (issue: {issue}57005[#57005])
```

File: docs/reference/release-notes/7.7.asciidoc [hunk 3]
```diff
@@ -1,21 +1,141 @@
...
 
+Authentication::
+* Expose idp.metadata.http.refresh for SAML realm {pull}56354[#56354]
+* Fix concurrent refresh of tokens {pull}55114[#55114] (issue: {issue}54289[#54289])
+
+CCR::
+* Retry follow task when remote connection queue full {pull}55314[#55314]
+
+Cluster Coordination::
+* fix the problem of recovering twice when perform a full cluster restart(#55564) {pull}55780[#55780] (issue: {issue}55564[#55564])
```
Comment (jrodewig) [inline docs/reference/release-notes/7.7.asciidoc:31]: Changing this to sentence case for consistency.
```suggestion
* Fix the problem of recovering twice when perform a full cluster restart(#55564) {pull}55780[#55780] (issue: {issue}55564[#55564])
```

File: docs/reference/release-notes/7.7.asciidoc [hunk 4]
```diff
@@ -1,21 +1,141 @@
...
+Recovery::
+* Fix trimUnsafeCommits for indices created before 6.2 {pull}57187[#57187] (issue: {issue}57091[#57091])
+
+SQL::
+* SQL: Fix unecessary evaluation for CASE/IIF {pull}57159[#57159] (issue: {issue}49672[#49672])
+* SQL: JDBC: fix access to the Manifest for non-entry JAR URLs {pull}56797[#56797] (issue: {issue}56759[#56759])
+* SQL: Fix JDBC url pattern in docs and error message {pull}56612[#56612] (issue: {issue}56476[#56476])
+* SQL: Fix serialization of JDBC prep statement date/time params {pull}56492[#56492] (issue: {issue}56084[#56084])
+* SQL: Fix issue with date range queries and timezone {pull}56115[#56115] (issue: {issue}56049[#56049])
+* SQL: SubSelect unresolved bugfix {pull}55956[#55956]
```
Comment (jrodewig) [inline docs/reference/release-notes/7.7.asciidoc:104]: Removes redundant prefixes.
```suggestion
* Fix unecessary evaluation for CASE/IIF {pull}57159[#57159] (issue: {issue}49672[#49672])
* JDBC: fix access to the Manifest for non-entry JAR URLs {pull}56797[#56797] (issue: {issue}56759[#56759])
* Fix JDBC url pattern in docs and error message {pull}56612[#56612] (issue: {issue}56476[#56476])
* Fix serialization of JDBC prep statement date/time params {pull}56492[#56492] (issue: {issue}56084[#56084])
* Fix issue with date range queries and timezone {pull}56115[#56115] (issue: {issue}56049[#56049])
* SubSelect unresolved bugfix {pull}55956[#55956]
```

File: docs/reference/release-notes/7.7.asciidoc [hunk 5]
```diff
@@ -1,21 +1,141 @@
...
+=== Upgrades
+
+Infra/Core::
+* Upgrade to Jackson 2.10.4 {pull}56188[#56188] (issue: {issue}56071[#56071])
+
+Infra/Packaging::
+* Upgrade bundled jdk to 14.0.1 {pull}57233[#57233]
+
+SQL::
+* SQL: update the JLine dependency to 3.14.1 {pull}57111[#57111] (issue: {issue}57076[#57076])
```
Comment (jrodewig) [inline docs/reference/release-notes/7.7.asciidoc:138]: Removes redundant prefix.
```suggestion
* Update the JLine dependency to 3.14.1 {pull}57111[#57111] (issue: {issue}57076[#57076])
```

File: docs/reference/release-notes/7.7.asciidoc [hunk 6]
```diff
@@ -1,21 +1,141 @@
...
+* Fix NPE in Partial Snapshot Without Global State {pull}55776[#55776] (issue: {issue}50234[#50234])
+* Fix Path Style Access Setting Priority {pull}55439[#55439] (issue: {issue}55407[#55407])
+
+[[regression-7.7.1]]
+[float]
+=== Regressions
+
+Infra/Scripting::
+* Change access pattern for GeoXXXValueSources {pull}54648[#54648]
+
```
Comment (jrodewig) [inline docs/reference/release-notes/7.7.asciidoc:126]: This PR was closed, not merged. I've removed the version labels.
```suggestion
```

**Review by jrodewig — APPROVED** [review]
Mostly LGTM. 

I removed a PR that was closed but not merged. I also removed several unneeded prefixes.

I'm going to incorporate those suggestions and merge this for release.

**Comment by jrodewig** [comment]
### Backport commits
7.x 6e67f1b
7.8 989c2ba
7.7 939dafd

**Comment by williamrandolph** [comment]
@jrodewig Wow — thanks for running with this! I really appreciate it.

**Pull request merged**